### ISSUE TITLE
when debugging an error occurs System.IO.FileNotFoundException:“Could not load file or assembly 'PresentationFramework

### ISSUE THREAD

**Issue by usernamedd** [dotnet/wpf#3290]
details below 
System.IO.FileNotFoundException:“Could not load file or assembly 'PresentationFramework, Version=42.42.42.42, Culture=neutral, PublicKeyToken=31bf3856ad364e35

 after building succefully , I add an project named wpfapp1 target netcore5.0 and then add references to other project in the wpf sln . Complie and run , it occurs that presentationFramework file could not loaded ,but the file is at the same directory with wpfapp1.exe

some suggest?

**Comment by ryalanms** [comment]
@usernamedd:  Which version of VS and the .NET SDK are you using?  Thanks.

**Comment by vatsan-madhavan** [comment]
Version=42,42,42,42 suggests that the user is using binaries from a dev-build. This is not any released version of .NET Core SDK

**Comment by usernamedd** [comment]
> @usernamedd: Which version of VS and the .NET SDK are you using? Thanks.

vs 2019 ,.net sdk version 5.0.100-preview.7.20366.6   ,

**Comment by usernamedd** [comment]
> Version=42,42,42,42 suggests that the user is using binaries from a dev-build. This is not any released version of .NET Core SDK

yes, but i have build by run build.cmd, how to get  a  released version ?

**Comment by vatsan-madhavan** [comment]
> yes, but i have build by run build.cmd, how to get a released version ?

From the SDK 😉 

I'm not saying you did anything wrong (or right). Since @ryalanms asked your SDK version, I figured this was a relevant hint for him.

**Comment by usernamedd** [comment]
now i have an another idea. delete sign info from each project then build and debug. the reason is that wpfapp1 exe probes static reference assembly from gac and finds the windowsbase.dll in gac ,but ,the windowsbase in gac is not the same version number as wpfapp1 expects .
